docs/tempo/website/guides/instrumentation.md

```diff
@@ -2,8 +2,15 @@
 title: Instrumentation Examples
 ---
 
-Check out these blog posts for help instrumenting your favorite languages to work with Tempo.
+Check out these resources for help instrumenting your favorite languages to work with Tempo.
 
-- [Java Spring Boot Auto-Instrumentation](https://grafana.com/blog/2021/02/03/auto-instrumenting-a-java-spring-boot-application-for-traces-and-logs-using-opentelemetry-and-grafana-tempo/)
-- [Go](https://grafana.com/blog/2020/11/09/trace-discovery-in-grafana-tempo-using-prometheus-exemplars-loki-2.0-queries-and-more/)
-- [.NET](https://grafana.com/blog/2021/02/11/instrumenting-a-.net-web-api-using-opentelemetry-tempo-and-grafana-cloud/)
+## Grafana Blog
+
+- [Java Spring Boot Auto-Instrumentation with OpenTelemetry](https://grafana.com/blog/2021/02/03/auto-instrumenting-a-java-spring-boot-application-for-traces-and-logs-using-opentelemetry-and-grafana-tempo/)
+- [Go OpenTelemetry + OpenMetrics Exemplars](https://grafana.com/blog/2020/11/09/trace-discovery-in-grafana-tempo-using-prometheus-exemplars-loki-2.0-queries-and-more/)
+- [.NET OpenTelemetry](https://grafana.com/blog/2021/02/11/instrumenting-a-.net-web-api-using-opentelemetry-tempo-and-grafana-cloud/)
```

Comment:
btw, all the examples use the OTEL instrumentation. Maybe we can add a section for OpenTelemetry instead of clarifying in each link the instrumentation library?

If someone adds <other> instrumentation examples, we can create new sections as required.